Automattic/jetpack
# Adds SVGR library to import svg files as React components in Jetpack extensions

## Proposed changes:

- Adds the [svgr library](https://react-svgr.com/) for Jetpack extensions to allow loading .svg files as React components
- Updates the Writing Prompt block to load its icon directly from the svg file

Note: I wasn't sure if this would be useful in the shared webpack configs, so I'm changing only the Jetpack plugin extensions config, for now.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/jetpack/pull/29450#pullrequestreview-1341942464

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Ensure your test site is connected to WP.com
* Enable experimental blocks (`define( 'JETPACK_EXPERIMENTAL_BLOCKS', true );`)
* Sandbox public-api and apply these changes to your wpcom sandbox with `bin/jetpack-downloader test jetpack add/svgr-for-icons` (if not, the block header and footer will be empty, but it works, otherwise).
* Insert a Writing Prompt block in a post. You should see the block icon in the sidebar (React component) and in the label within the block (svg file).
* Save the block and ensure you can see the icon on the front-end, within the post content, as well.
